### PR TITLE
Fixes #16103: Add a CIS category and put everything in it

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/16103

### CODE REVIEW

File: cis/mk.sh [hunk 1]
```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
```
Comment (Fdall) [inline cis/mk.sh]: You can remove this file

File: cis/packaging/rudder-synchronize [hunk 2]
```diff
@@ -135,23 +135,16 @@ def import_technique(path):
     try:
         with open(path) as f:
             data = json.load(f)["data"]
-        # Call the technique API
-        techniquesEndpoint = RudderEndPoint("https://localhost/ncf", TOKEN, verify=False)
-        techniqueData = { "path": "/var/rudder/configuration-repository/ncf", "technique": data, "methods": [] }
-        response = techniquesEndpoint.request("PUT", "/api/techniques", None, techniqueData, return_raw=True)
-
         # Extend the technique json with GM
+        data["internalId"] = "toto"
```
Comment (Fdall) [inline cis/packaging/rudder-synchronize]: ?

File: cis/packaging/postinst [hunk 3]
```diff
@@ -16,7 +42,10 @@ git commit -m "CIS plugin installation"
 
 # Import Techniques
 ${CONFIGURATION_PATH}/rudder-synchronize import technique ${CONFIGURATION_PATH}/techniques
-rudder server reload-techniques
+git add techniques/CIS_plugin
+git commit -m "add techniques"
```
Comment (Fdall) [inline cis/packaging/postinst]: I think you can improve that commit message

File: cis/packaging/postinst [hunk 4]
```diff
@@ -16,7 +42,10 @@ git commit -m "CIS plugin installation"
...
+curl -H "X-API-TOKEN:  $(cat /var/rudder/run/api-token)" http://localhost/rudder/api/latest/system/reload/techniques -X POST -k
+#rudder server reload-techniques
```
Comment (Fdall) [inline cis/packaging/postinst]: Not needed anymore

File: cis/packaging/postinst [hunk 5]
```diff
@@ -1,12 +1,38 @@
 #!/bin/bash
 CONFIGURATION_PATH=/var/rudder/packages/rudder-plugin-cis
+FOLDERS="ncf/30_generic_methods/cis techniques/CIS_plugin"
```
Comment (Fdall) [inline cis/packaging/postinst]: Why put a space in the name?

Comment (VinceMacBuche) [reply cis/packaging/postinst]: no space it is two separated folders, used in for loop

File: cis/packaging/postinst (hunk 4 above)
Comment (VinceMacBuche) [reply cis/packaging/postinst]: In fact it should be reverse ;)

File: cis/packaging/rudder-synchronize (hunk 2 above)
Comment (VinceMacBuche) [reply cis/packaging/rudder-synchronize]: Yes, maybe a fixed uuid ;)

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
PR rebased

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Pull request merged**